forLoopTimeout/forLoopTimeout.js

```diff
@@ -4,7 +4,7 @@
 // The output should be 1, 2, 3, .... 10. Right now it just prints 11.
 // I've been asked this three times in separate interviews.
 
-for (var i = 1; i <= 10; i++) {
+for (let i = 1; i <= 10; i++) {
```

Comment:
Great! Now, to challenge you further.. how would you solve this without es6 usage